docs/modules/install/pages/manual.adoc

```diff
@@ -12,7 +12,7 @@ In order to develop on decidim, you will need:
 
 We are starting with an Ubuntu 22.04.1 LTS. This is an opinionated guide, so you are free to use the technology that you are most comfortable. If you have any doubts and you are blocked you can go and ask on https://matrix.to/#/#decidimdevs:matrix.org[our Matrix.org chat room for developers].
 
-We recommend to have at least some basic proficiency in Ruby on Rails (a good starting point is http://guides.rubyonrails.org/getting_started.html[Getting Started with Ruby on Rails]) and have some knowledge on how gems work.
+We recommend to have at least some basic proficiency in GNU/Linux (i.e. how to use the command-line, packages, etc), networking knowledge, server administration, Ruby on Rails development basics (a good starting point is http://guides.rubyonrails.org/getting_started.html[Getting Started with Ruby on Rails]) and have some knowledge on how gems work.
```

Comment:
Hey, @andreslucena, i have played a bit with the text. We need to cover also the npm stuff, as there may be issues with that as well (#11792 came to my mind) Feel free to iterate on my suggestion :) 

```suggestion
We recommend to have at least some basic proficiency in GNU/Linux (i.e. how to use the command-line, packages, etc), networking knowledge, server administration, development in general, and some basic knowledge about software package managers. It would be great to have Ruby on Rails development basics (a good starting point is http://guides.rubyonrails.org/getting_started.html[Getting Started with Ruby on Rails]) and have some knowledge on how package libraries are working ( we use `bundler` for got handling ruby packages, and `npm` / `yarn` for handling javascript). 
```

Reply:
Just removed an extra "got" in your text and some extra spacings... LGTM